src/arrangement.cpp

```diff
@@ -176,9 +182,8 @@ void init_arrangement(py::module &m) {
         }, py::keep_alive<0, 1>())
         .def("insert_non_intersecting_curve", &insert_non_intersecting_curve_in_arr)
         .def("insert_non_intersecting_curves", &insert_non_intersecting_curves_in_arr)
-        .def("insert", [](Segment_Arrangement_2& self, Segment_2& segment) {
-            CGAL::insert(self, segment);
-        })
+        .def("insert", py::overload_cast<Segment_Arrangement_2&, Segment_2&>(&insert_segment_in_arr))
```

Comment:
Do we really need py::overload_cast? 
The two functions have different names, so I think it's fine to just use the pointer to the function, without the cast.

Reply:
You're right, checked it on my code